app/models/order.rb

```diff
@@ -12,4 +12,8 @@ class Order < ActiveRecord::Base
   def total_price
     items.inject(0) { |sum, item| sum + item.price }
   end
+
+  def line_item_price(item, order)
+    item.price * quantity_of_item(item, order)
+  end
```

Comment:
I think we abstracted this away to the `application_controller.rb` file. It should be accessible from there as a helper method.